MetaMask/metamask-extension
# feat: add Portfolio button to BTC accounts

## **Description**

Readds the "Portfolio" button for non-EVM accounts

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/28184?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/pull/26148 https://github.com/MetaMask/metamask-extension/issues/28185

## **Manual testing steps**

1. Go to the Wallet page
2. Switch to a BTC account
3. The Portfolio button should be there

## **Screenshots/Recordings**



### **Before**

<img width="142" alt="before" src="https://github.com/user-attachments/assets/2ff13df2-7db1-44e3-a36c-b9a85248b3c1" />


### **After**

<img width="142" alt="Screenshot 2025-01-17 at 14 50 46" src="https://github.com/user-attachments/assets/6c65ad87-1822-4fb1-a07b-31dde9066f52" />


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
